src/index.js

```diff
@@ -0,0 +1,130 @@
+// Constants
+const BASE_URL = 'http://127.0.0.1:5000';
+const DEFAULT_CITY = 'Los Angeles';
+const increaseTemp = document.getElementById('increaseTempControl');
+const decreaseTemp = document.getElementById('decreaseTempControl');
+const tempValue = document.getElementById('tempValue');
+const landscape = document.getElementById('landscape');
+const cityNameInput = document.getElementById('cityNameInput');
+const cityNameReset = document.getElementById('cityNameReset'); 
+const headerCityName = document.getElementById('headerCityName');
+const currentTempButton = document.getElementById('currentTempButton');
+
+// Changes color + landscape by Temperature
+let temp = 60;
+const changeTemp = () => {
+    let landscapeEmojis = '';
+    if (temp >= 80) {
+        tempValue.style.color = 'red';
+        landscapeEmojis = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+    } else if (temp< 80 && temp >= 70) {
+        tempValue.style.color = 'orange';
+        landscapeEmojis = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+    } else if (temp < 70 && temp >= 60) {
+        tempValue.style.color = 'yellow';
+        landscapeEmojis = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+    } else if (temp < 60 && temp >= 50) {
+        tempValue.style.color = 'green';
+        landscapeEmojis = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+    } else {
+        tempValue.style.color = 'teal';
+        landscapeEmojis = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+    }
```

Comment:
Long chains of conditionals like this are repetitive, brittle, and error prone. It would be easy to introduce a typo in one or more of the checks, and if we need to make changes, adding an additional check requires quite a bit of awareness about what the other branches are doing. Instead, consider storing the checks in a data structure that can be iterated over. We could use a list like:
```js
const temperatureColors = [
    [80, "red", "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂"],
    [70, "orange", "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷"],
    [60, "yellow", "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃"],
    [50, "green", "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲"],
    [40, "teal", "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲"],
];
```
By arranging the temperatures in order, we can iterate through the list and compare the temperature to the value in the list to see if the list value is below our value, in which case we should use that class name. If we reach the end without a match, we could use the final value, or apply some other logic.